packages/plugins/experimental/plugin-transcription/src/components/Transcript/Transcript.tsx

```diff
@@ -2,68 +2,237 @@
+                  ? {
+                      readonly: true,
+                      accessoryHtml: `<span class="pbs-1 text-xs text-neutral-500">${formatElapsed(start, end)}</span>`,
+                    }
+                  : {}
+              ) satisfies DxGridCellValue;
+
+              cells[toPlaneCellIndex({ col: 1, row })] = (
+                segmentIndex < 0
+                  ? {
+                      value: author,
+                      readonly: true,
+                      // TODO(burdon): Color based on username.
+                      className: authorClasses,
+                    }
+                  : {
+                      readonly: true,
+                      accessoryHtml: `<div class="${segmentTextClasses}">${text.trim()}</div>`,
+                    }
+              ) satisfies DxGridCellValue;
+            }
+
+            return cells;
+          }
+          default:
+            return {};
+        }
+      };
+    }
+  }, [dxGrid, queueMap, blocks]);
 
-          {block.segments.map((segment, i) => (
-            <div key={i} className='group flex flex-col'>
-              <div className='px-2'>{segment.text}</div>
-              <div className='flex gap-1 items-center justify-between px-2'>
-                <div className='grow' />
-                <div className='truncate text-xs text-subdued'>
-                  {formatDistanceToNow(segment.started, { addSuffix: true })}
-                </div>
-                <IconButton
-                  disabled
-                  icon='ph--bookmark-simple--regular'
-                  label={t('bookmark button')}
-                  iconOnly
-                  size={4}
-                  classNames={hoverButton}
-                />
-              </div>
+  const handleWheel = useCallback(
+    (event: WheelEvent) => {
+      if (!ignoreAttention && !hasAttention) {
+        event.stopPropagation();
+      }
+    },
+    [hasAttention, ignoreAttention],
+  );
+
+  return (
+    <div role='none' ref={ref} className='grow'>
+      {transcriptColumns && (
+        <>
+          <div className='relative'>
+            <div
+              className='absolute top-0 z-10 p-1 border invisible'
+              style={{ left: timeWidth, width: transcriptColumns.grid['1'].size }}
+            >
+              <div ref={measureRef} className={mx(segmentTextClasses)} />
             </div>
-          ))}
-        </div>
-      ))}
+          </div>
+          <Grid.Root id={`${attendableId}--transcript`}>
+            <Grid.Content
+              ref={setDxGrid}
+              className='[--dx-grid-base:var(--dx-baseSurface)] [--dx-grid-lines:var(--dx-baseSurface)] [&_.dx-grid]:min-bs-0 [&_.dx-grid]:select-auto'
+              columns={transcriptColumns}
+              rowDefault={transcriptRows}
+              rows={rows}
+              limitColumns={2}
+              limitRows={queueMap.length}
+              onWheel={handleWheel}
+            />
+          </Grid.Root>
```

Comment:
While this was a productive experiment, ultimately I think we should probably just use CodeMirror for this.